lib/ansible/modules/cloud/azure/azure_rm_securitygroup.py

```diff
@@ -545,6 +548,11 @@ def __init__(self):
                                                    supports_check_mode=True)
 
     def exec_module(self, **kwargs):
+        self.client = self.get_mgmt_svc_client(NetworkManagementClient)
+        # tighten up poll interval for security groups; default 30s is an eternity
+        # this value is still overridden by the response Retry-After header (which is set on the initial operation response to 10s)
+        self.client.config.long_running_operation_timeout = 3
+        self.nsg_models = self.client.network_security_groups.models
```

Comment:
Since we may use the model out of a module class (e.g. instance_to_dict() functions usually be defined out of the class, but it may use SDK model), how to handle such scenario?

Reply:
@yuwzho can you point to an example? I think I partially understand what you're getting at, but want to make sure...

Reply:
If it helps, note that every models have a `as_dict` method:
https://github.com/Azure/msrest-for-python/wiki/msrest-0.4.12---Serialization-change